.github/workflows/tests.yml

```diff
@@ -8,58 +8,18 @@ env:
 jobs:
   lint:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.8]
-
     steps:
       - uses: actions/checkout@v2
-
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v1
-        with:
-          python-version: ${{ matrix.python-version }}
-
-      - name: Set up Node/yarn
-        uses: actions/setup-node@v1
-        with:
-          node-version: "16.x"
-
-      - name: Cache python wheels
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: |
-            ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('setup.py', 'docs/requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-${{ matrix.python-version }}-
-            ${{ runner.os }}-pip-
-
-      - name: Cache node_modules
-        uses: actions/cache@v2
-        with:
-          path: "node_modules"
-          key: |
-            ${{ runner.os }}-node-modules-${{ hashFiles('yarn.lock') }}
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip setuptools wheel
-          python -m pip install --upgrade pre-commit
-          python -m pip install -e .
-          yarn --frozen-lockfile
-
-      - name: Lint
-        run: |
-          pre-commit run --all-files
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.3
 
   # Build docs on a number of Python versions. In the future this can be
   # where tests go.
   tests:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
I just want to highlight that we're dropping Python 3.6 support here, in case anybody has a strong objection. I think it is fine because it will no longer have official support soon anyway, but just making it clear here.